hw/chisel/src/main/scala/snax/xdma/xdmaFrontend/DMACtrl.scala

```diff
@@ -187,13 +187,15 @@ class DMACtrl(
   val i_csrmanager = Module(
     new CsrManager(
       csrNumReadWrite = 2 + // Reader Pointer needs two CSRs
-        readerparam.rwParam.agu_param.dimension * 2 + // Strides + Bounds for read
+        readerparam.rwParam.agu_param.dimension * 2 + // Strides + Bounds for reader
```

Comment:
is the mask by default all 1's, or do we need to program this explicitly?

Reply:
We have to program it explicitly. I reuse CSRManager here, and it does not support default values.